user/timepix3/module/src/Timepix3Producer.cxx

```diff
@@ -1000,7 +991,7 @@ void Timepix3Producer::RunLoop() {
   ts_thread.join();
   ts_thread.~thread();
   // show header statitstics
-  //std::cout << "Headers: " << listVector(header_counter) << endl;
+  std::cout << "Headers: " << listVector(header_counter) << endl;
```

Comment:
This could be also a log message of some level?

Reply:
yeah, but it's of limited use and will probably just confuse users. I want to look at it for a while and then either promote it or remove it.